README.md

```diff
@@ -8,7 +8,7 @@
 
         git clone git@github.com:eaudeweb/scratch.git
 
-1. Copy docker example files:
+2. Copy docker example files:
```

Comment:
Hardcoded numbered lists are a bit harder to maintain. Any particular reason why you made this change? When you use "1." everywhere, the markdown renderer knows how to display the list properly. Checking the old README I see the numbering is broken, but that's probably because of indentation problems. If the indentation is fixed, the numbering should also be fixed.